app/celery/letters_pdf_tasks.py

```diff
@@ -103,17 +104,33 @@ def get_pdf_for_templated_letter(self, notification_id):
 @notify_celery.task(bind=True, name="update-billable-units-for-letter", max_retries=15, default_retry_delay=300)
 def update_billable_units_for_letter(self, notification_id, page_count):
     notification = get_notification_by_id(notification_id, _raise=True)
-
-    billable_units = get_billable_units_for_letter_page_count(page_count)
-
-    if notification.key_type != KEY_TYPE_TEST:
-        notification.billable_units = billable_units
+    if page_count > 10:
```

Comment:
This approach was the simplest way to make the change so that we can mark a letter as invalid. 
1. the name of the method can change and a comment can be made. 
2. I thought about that and it is a very complex change compared to this approach.
The first row is validated in the admin app, but it was deemed to lengthy to validate each row in the CSV.

The page count can be put in the config.

Reply:
```python
from notifications_utils import LETTER_MAX_PAGE_COUNT
```
😉

Reply:
Thanks @quis I did find it

Reply:
If we get an API to send letters to the post provider, then we can refactor this work flow. I think for now let's go with this since it works, doesn't change the user experience, and can be implemented today.